arithmoi.cabal

```diff
@@ -59,13 +59,15 @@ library
     Math.NumberTheory.ArithmeticFunctions.Moebius
     Math.NumberTheory.ArithmeticFunctions.SieveBlock
     Math.NumberTheory.Curves.Montgomery
+    Math.NumberTheory.DirichletCharacters
     Math.NumberTheory.Euclidean
     Math.NumberTheory.Euclidean.Coprimes
     Math.NumberTheory.Moduli
     Math.NumberTheory.Moduli.Chinese
     Math.NumberTheory.Moduli.Class
     Math.NumberTheory.Moduli.DiscreteLogarithm
     Math.NumberTheory.Moduli.Equations
+    Math.NumberTheory.Moduli.Internal
```

Comment:
I believe internals are not intended to be publicly exposed. Please move them under `other-modules` section.

Reply:
Thanks for this catch!